docs/csharp/language-reference/operators/arithmetic-operators.md

```diff
@@ -11,6 +11,9 @@ f1_keywords:
   - "%_CSharpKeyword"
   - "+_CSharpKeyword"
   - "-_CSharpKeyword"
+  - "%=_CSharpKeyword"
+  - "*=_CSharpKeyword"
+  - "_CSharpKeyword"
```

Comment:
This one looks strange to me. Based on my understanding, [this line](https://github.com/dotnet/roslyn/blob/master/src/VisualStudio/CSharp/Impl/LanguageService/CSharpHelpContextService.cs#L50) in Roslyn shouldn't have `text` being empty. There should be something before the underscore I think.

Reply:
In this case, I think the language service is not picking up anything where the user's cursor is. I think it's good to have a fallback page so they at least land in the vicinity of where they might find help. But it might be better to add this keyword here instead: https://github.com/dotnet/docs/blob/master/docs/csharp/language-reference/operators/index.md. Thoughts @BillWagner ?

Reply:
If Roslyn generates `_CSharpKeyword` without anything before the underscore when the cursor isn't on a keyword or something like that, I think it should be fixed in Roslyn not here. Nothing before the underscore is meaningless, and Roslyn shouldn't generate it like that.

@gewarren Do you know who from Roslyn team we can ask?

Reply:
@Youssef1313 I don't know who to contact in the Roslyn team. You could file an issue there. Either way, I don't think you can guarantee that Roslyn will always capture text, and it's easy and doesn't hurt to add this here.

Reply:
Then probably the c# landing page would be a better place?

Reply:
I put the keyword at the root of the language reference docs, so /language-reference/index.md.